locale/ar.json

```diff
@@ -560,7 +560,7 @@
   },
   "src_dot_back": {
     "context": "button",
-    "string": "Back"
+    "string": "عودة"
```

Comment:
Why there are translation changes in this PR?

Reply:
because of the mistaken transifex push